_source/_includes/log-shipping/multiline-logs-filebeat.md

```diff
@@ -0,0 +1,81 @@
+## Multiline logs
+
+
+If your original logs span multiple lines, you may find that they arrive in your Logz.io account split into several partial logs. You have the option to use either an explicit configuration or an annotation to configure Filebeat to concatenate multiline logs and avoid having them separated into multiple log documents.
+
+You will need to provide a regex expression that can identify the first log in a multiline log. Filebeat will concatenate the log lines that follow until it identifies the next log that matches the regex expression. In other words, there is no explicit regex expression to demarcate the _end_ of a multiline log.
+
+This behavior is managed differently depending on your deployment method:
+
+* If you are using autodiscover hints & annotations, add an annotation to your deployment.
+* If you are not using autodiscover, use an explicit configuration.
+
+
```

Comment:
There's a major benefit of using the hints system here, and I think we should mentions it, so it maybe will help them better to consider whether they want to use filebeat with or without autodiscover:
if they use hints, they can manage the regexes for each component separately, not having to create a regex that will match all the possible cases that they need, like they would need to do when using the regular config without autodiscover.
For example, if i have one pod that I know will need regex X, and another that will need regex Y- in the regular config i'll have to create a regex that can "catch" both X and Y, and if I have another pod with regex Z, i'll need to reconfigure filebeat. But if i'm using hints & annotations, i'll just have to add to pod X annotation with regex X, to pod Y annotation with regex Y, etc.. without needing to change anything on Filebeat itself.

Reply:
addressed.